<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>566</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTabWidget" name="tabWidgetSideBar">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
      </widget>
      <widget class="QSplitter" name="splitterRight">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>0</number>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QTabWidget" name="tabWidgetCave">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidgetOutput">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewProject"/>
     <addaction name="actionNewIrbis"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionRecentProjects"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFileAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="actionCloseOther"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowSidebar"/>
    <addaction name="actionShowOutput"/>
    <addaction name="actionClearOutput"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionUnitBuilder"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionExpression"/>
    <addaction name="actionFunction"/>
    <addaction name="actionModule"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdd"/>
   <addaction name="menuBuild"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project...</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save File As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionShowSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Sidebar</string>
   </property>
   <property name="shortcut">
    <string>Alt+0</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionCloseOther">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Other</string>
   </property>
  </action>
  <action name="actionUnitBuilder">
   <property name="text">
    <string>UnitBuilder...</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionNew_Irbis_File">
   <property name="text">
    <string>New Irbis File...</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionOpen_Irbis_File">
   <property name="text">
    <string>Open Irbis File...</string>
   </property>
  </action>
  <action name="actionRecentProjects">
   <property name="text">
    <string>Recent Projects</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>Project...</string>
   </property>
  </action>
  <action name="actionNewIrbis">
   <property name="text">
    <string>Irbis File...</string>
   </property>
  </action>
  <action name="actionIrbis_Library">
   <property name="text">
    <string>Irbis Library...</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRecentFiles">
   <property name="text">
    <string>Recent Files</string>
   </property>
  </action>
  <action name="actionShowOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Output</string>
   </property>
  </action>
  <action name="actionClearOutput">
   <property name="text">
    <string>Clear Output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionExpression">
   <property name="text">
    <string>Expression</string>
   </property>
  </action>
  <action name="actionFunction">
   <property name="text">
    <string>Function</string>
   </property>
  </action>
  <action name="actionModule">
   <property name="text">
    <string>Module</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
